crates/solidity/outputs/npm/build/src/main.rs

```diff
@@ -17,12 +17,20 @@ use solidity_language::{GrammarConstructorDslV2, SolidityDefinition};
 fn main() -> Result<()> {
     // Generate files in the source crate:
     {
-        let lang_def = SolidityDefinition::create();
-        let grammar = Grammar::from_dsl_v2(&lang_def);
+        let language = SolidityDefinition::create();
+        let grammar = Grammar::from_dsl_v2(&language);
+        let ast_model = AstModel::create(&language);
 
-        let crate_dir = CargoWorkspace::locate_source_crate("solidity_npm_crate")?;
+        CodeGenerator::write_backend(
+            &grammar,
+            &ast_model,
+            &CargoWorkspace::locate_source_crate("solidity_npm_crate")?.join("src/generated"),
+        )?;
 
-        CodeGenerator::write_source(&crate_dir.join("src/generated"), &grammar)?;
+        CodeGenerator::write_frontend(
```

Comment:
I'd like to change the backend/frontend naming here as well since it's slightly confusing, but let's do that in a follow-up.